apps/next/src/components/Header/components/HeaderActions.tsx

```diff
@@ -26,7 +28,10 @@ export const HeaderActions: FC<Props> = ({ account }) => {
       <IconButton
         disabled={!account}
         size="small"
-        onClick={() => history.push(`/receive?accId=${account?.id}`)}
+        onClick={() => {
+          capture('TokenReceiveClicked');
```

Comment:
Legacy event includes the `chainId` parameter here. Maybe it would be a good idea to have an event for when users change the network on the Receive page?

<img width="306" height="350" alt="Image" src="https://github.com/user-attachments/assets/d3a00b8f-4494-499d-860a-37d60f795dee" />